wp-admin/css/colors/_mixins.scss

```diff
@@ -40,4 +40,12 @@
 		border-color: darken( $button-color, 15% ) !important;
 		text-shadow: none !important;
 	}
+
+	&.button-hero {
+		box-shadow: 0 2px 0 darken( $button-color, 15% ) !important;
+		&:active {
+		 	box-shadow: inset 0 3px 0 darken( $button-color, 15% ) !important;
```

Comment:
!important should not be used<br>Expected 0 spaces between parentheses instead of 1